es6-template-literals/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Javascript ES6 - Template Literals</title>
+</head>
+
+<body>
+  <script>
+    const number = 9200;
+    const street = ' Irvine Center Drive';
+    const address = `${number} ${street}`;
+
+    console.log(`LearningFuze: ${address}`);
+
+    const firstName = 'Uzair';
+    const lastName = 'Ashraf';
+    const age = 25;
+
+    const bio = `My name is  ${firstName}  ${lastName} and I am ${age} years old.`;
+
+    console.log(`Bio: ${bio}`);
+
+    const fruit = 'apple';
+    const item = 'pen';
+
+    const applePen = `${fruit}${item}`;
+
+    console.log(`I have a ${item}`);
+    console.log(`I have an ${fruit}`);
```

Comment:
Passing `console.log` a single string template argument is not the same operation as passing it two arguments:

---

![image](https://user-images.githubusercontent.com/16143771/139779063-2172495e-a3a5-4b5c-9902-d896f073241d.png)